setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(
+    name='Image DePHI',
```

Comment:
```suggestion
    name='ImageDePHI',
```
This shouldn't have spaces, since it's used to build the package file name.